predbat/Dockerfile.standalone

```diff
@@ -13,12 +13,16 @@ RUN apt-get install -y python3-venv
 RUN apt-get install -y python3-requests
 RUN apt-get install -y python3-yaml
 #RUN apt-get install -y python3-asyncio
-RUN apt-get install -y python3-aiohttp
 RUN apt-get install csh
-RUN apt-get install -y python3-tz
+RUN apt-get install -y python3-aiohttp
+RUN apt-get install -y python3-tz 
+RUN apt-get install -y vim wget zip 
 
-WORKDIR /config
-COPY rootfs /config
+WORKDIR /wkspace
+COPY rootfs /wkspace
+RUN wget https://codeload.github.com/springfall2008/batpred/zip/refs/heads/main -O main.zip
+RUN unzip main.zip
+RUN cp batpred-main/apps/predbat/predbat.py predbat.py 
```

Comment:
Should this be *.py?

Reply:
yes, with your split design, it should be *py